src/Microsoft.AspNetCore.Authentication.OpenIdConnect/OpenIdConnectHandler.cs

```diff
@@ -585,6 +578,11 @@ protected virtual Task<bool> HandleSignOutCallbackAsync()
                         tokenEndpointResponse = await RedeemAuthorizationCodeAsync(tokenEndpointRequest);
                     }
 
+                    if (!string.IsNullOrEmpty(tokenEndpointResponse.Error))
```

Comment:
I see. It's in the OAuth spec. Thanks for pointing out. I'll make the change to the flow accordingly.